lib/recurly/xml/nokogiri.rb

```diff
@@ -46,7 +46,7 @@ def text= text
       end
 
       def to_s
-        root.to_xml(:indent => 0).gsub(/$\n/, '')
+        root.to_xml(:indent => 0,:save_with => Nokogiri::XML::Node::SaveOptions::NO_EMPTY_TAGS).gsub(/$\n/, '')
```

Comment:
Silly Jruby-1.9. All other ruby flavors in the matrix generate an empty tag (a tag with no content) as follows ```<element></<element>```. Jruby 1.7.19 (against a floating nokogiri) generates ```</element>```. When checked against the expectation it does not match because jruby 1.7.19 is using self closing tags. This directive tells nokogiri to not generate self closing tags.